src/integrationtests/resources/com/aws/greengrass/integrationtests/deployment/FleetConfigWithSimpleAppv1AndGreenSignal.json

```diff
@@ -0,0 +1,21 @@
+{
```

Comment:
You don't need the new required capability in the deployment doc for the tests to pass?

Reply:
I am adding that in the test code. Check line `307` of `SubGroupDeploymentIntegrationTest`